### PR TITLE
[UPGRADE] Android SDK and dependencies to support SDK 30

### PR DESCRIPTION
**Pull request by sethkfman**
**Description**

Bump Android SDK version to compile and target 30. Also, updated dependencies required for this bump. 

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Resolves #???

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -264,6 +264,10 @@ dependencies {
 	androidTestImplementation('com.wix:detox:+') { transitive = true }
 	androidTestImplementation 'junit:junit:4.12'
 	implementation "androidx.swiperefreshlayout:swiperefreshlayout:1.0.0"
+	implementation "com.squareup.okhttp3:okhttp:4.9.1"
+    implementation "com.squareup.okhttp3:logging-interceptor:4.9.1"
```
Comment (rickycodes) [inline android/app/build.gradle:268]: tabbing is inconsistent here

File: app/components/Views/Entry/index.js [hunk 2]
```diff
@@ -77,7 +77,7 @@ const Entry = (props) => {
 	const animation = useRef(null);
 	const animationName = useRef(null);
 	const opacity = useRef(new Animated.Value(1)).current;
-
+	console.log('Hermes Enabled', !!global.HermesInternal);
```
Comment (rickycodes) [inline app/components/Views/Entry/index.js:80]: you'll want to take this out.

**Review by rickycodes — CHANGES_REQUESTED** [review]
this looks good to me and no longer crashes woot

just a few small things. awesome stuff.

**Review by rickycodes — APPROVED** [review]
LGTM

**Review by cortisiko — APPROVED** [review]
🌮 🌮 🌮 🌮

**Pull request merged**